### PR TITLE
Stress-tests

### PR DESCRIPTION
**Pull request by ustulation**
This change is [Reviewable](https://reviewable.io/reviews/maidsafe/safe_core/331)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @krishnaIndia

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Comment by ustulation** [comment]
Reviewed 3 of 4 files at r1, 3 of 3 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/safe_core/331)*

File: examples/email_stress_test.rs [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+
+    // ------------------------------------------------------------------------
+    // Create bots
+    let mut now = Instant::now();
+    let bots: Vec<_> = (0..BOTS).map(|n| Bot::new(n, &mut rng, args.flag_get_only)).collect();
+    let mut duration = now.elapsed();
+    info!("Create accounts for {} bots: {} secs, {} millis\n",
+          BOTS,
+          duration.as_secs(),
+          duration.subsec_nanos() / 1000000 / 1000000);
```
Comment (madadam) [inline examples/email_stress_test.rs]: One too many divisions by 1000000 here, I think.

**Comment by ustulation** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/safe_core/331)*

File: examples/email_stress_test.rs (hunk 1 above)
Comment (ustulation) [inline examples/email_stress_test.rs]: you are right - will correct this

Comment (ustulation) [inline examples/email_stress_test.rs]: fixed in the upcoming PR

**Pull request merged**